katalog/gatekeeper/core/kustomization.yaml

```diff
@@ -23,4 +23,4 @@ resources:
 images:
   - name: openpolicyagent/gatekeeper
     newName: registry.sighup.io/fury/openpolicyagent/gatekeeper
-    newTag: v3.12.0
+    newTag: v3.13.3
```

Comment:
Shouldn't this be 3.14.x?

Reply:
this wasn't changed in this PR, we are seeing this change because this PR was branched from #100 's branch and has not been rebased after the changes done over there.

please @smerlos rebase this PR

Reply:
done